akka-http/client/src/main/scala/endpoints/akkahttp/client/Urls.scala

```diff
@@ -31,25 +34,28 @@ trait Urls extends algebra.Urls {
       }
     }
 
-  def qs[A](name: String, docs: Documentation)(implicit value: QueryStringParam[A]): QueryString[A] =
-    a => Some(s"$name=${value.apply(a)}")
+  def qs[A](name: String, docs: Documentation)(implicit param: QueryStringParam[A]): QueryString[A] =
+    a => {
+      val params = param(a)
+      if (params.isEmpty) None
+      else Some(params.map(v => s"$name=$v").mkString("&"))
```

Comment:
Nothing wrong here, just for the style I usually find nicer to use pattern matching:
``param(a) match {
  case Nil => None
  case params => Option(params.map(v => s"$name=$v").mkString("&"))
}
```